packages/create-vite/README.md

```diff
@@ -55,6 +55,8 @@ Currently supported template presets include:
 - `lit-element-ts`
 - `svelte`
 - `svelte-ts`
+- `solid-ts`
```

Comment:
No double `solid-ts` :sweat_smile: 

```suggestion
- `solid`
```

Reply:
Threre are two template added, first for js, second for typescript.